libs/blocks/graybox-promote/components/graybox-promote-lit.js

```diff
@@ -216,20 +213,21 @@ class GrayboxPromote extends LitElement {
         ref,
         owner,
         url,
-        spData
+        spData,
       }) => {
         try {
-          const {root, gbRoot, driveId} = spData;
+          const { root, gbRoot, driveId } = spData;
           const mainRepo = repo.replace('-graybox', '');
           const promote = await fetch(`${promoteUrl}?spToken=${this.spToken}&projectExcelPath=${getProjectExcelPath(url)}&rootFolder=${root}&gbRootFolder=${gbRoot}&experienceName=${experienceName}&adminPageUri=${`https://milo.adobe.com/tools/graybox?ref=${ref}%26repo=${mainRepo}%26owner=${owner}`}%26host=business.adobe.com%26project=${mainRepo.toUpperCase()}%26referrer=MOCK_REF&draftsOnly=${promoteDraftsOnly}&promoteIgnorePaths=${promoteIgnorePaths}&driveId=${driveId}&ignoreUserCheck=true`);
           const promoteRes = await promote.json();
           if (promoteRes?.code === 200) {
             return 'Successfully promoted';
-          } else {
-            throw new Error('Could not promote. Please try again.');
           }
+          throw new Error('Could not promote. Please try again.');
         } catch (e) {
+          // eslint-disable-next-line no-console
           console.log(e);
+          return 'Failed to promote.';
```

Comment:
On L226, an error is returned, shouldn't this be the case here as well?

Reply:
Actually I don't need anything to be returned if a error is threw, the lit component will handle it. I added the return to avoid the eslint error saying that something should be returned.